src/libAtomVM/context.c

```diff
@@ -971,3 +971,175 @@ term context_get_monitor_pid(Context *ctx, uint64_t ref_ticks, bool *is_monitori
     }
     return term_invalid_term();
 }
+
+int context_get_catch_label(Context *ctx, Module **mod)
```

Comment:
should I assume this is a copy and paste of existing code? or should I check for other changes?